gap/almostsimple.gi

```diff
@@ -906,8 +918,10 @@ RECOG.HomFDPM := function(data,x)
 end;
 
 #! @BeginChunk AltSymBBByDegree
-#! TODO
+#! This method is a black box constructive (?) recognition of alternating
+#! and symmetric groups.
```

Comment:
Is there a paper ref? You might want to ask Alice for one

Reply:
Beals, Robert & Leedham-Green, Charles & Niemeyer, Alice & Praeger, Cheryl & Seress, Ákos. (2005). Constructive recognition of finite alternating and symmetric groups acting as matrix groups on their natural permutation modules. Journal of Algebra - J ALGEBRA. 292. 4-46. 10.1016/j.jalgebra.2005.01.035.

Reply:
I have added this to the bibliography in PR #106, with the key `BLGN+05`. Therefore, in this bit of documentation, it would be great if you could include refer to the paper in a sentence that includes `<Cite Key="BLGN+05"/>`.

Reply:
(Although then your documentation won't compile until #106 is merged.)

Reply:
There is now a reference in the lines below